in_text/text_01_spatial.yml

```diff
@@ -33,16 +33,16 @@ entities:
     data-name: study_stream_reaches.zip
     data-description: >-
       GIS polylines data for the 70 stream segments included in the upper Delaware River Basin that
-      have been included in this study. Stream reaches include those on the East and West Branch of the 
+      have been included in this study. Stream reaches include those on the East and West Branch of the
       Delaware River, mainstem Delaware River reaches above Lordville, New York, and the Neversink River.
       The stream network is originally derived from the USGS Geospatial Fabric v1.
     attributes:
     -
       attr-label: subsegid
       attr-def: >-
-        An alternative unique reach identifier which was created in order to split segments such that 
+        An alternative unique reach identifier which was created in order to split segments such that
         segment start and endpoints were connected by one reach with no intersecting reaches. The ID was created
-        using the seg_id from the Geospatial Fabric and an index from one to the number of splits (e.g., {subseg_id}_{index}). 
+        using the seg_id from the Geospatial Fabric and an index from one to the number of splits (e.g., {seg_id}_{index}). 
```

Comment:
changing to align with first data release change - want to improve clarity because this is the seg_id from GF you just mentioned